controllers/profiles/reconciler_dev.go

```diff
@@ -254,28 +266,44 @@ func getDeploymentFailureReasonOrDefaultReason(deployment *appsv1.Deployment) st
 	return failure
 }
 
-// mountWorkflowDefConfigMapMutateVisitor mounts the given ConfigMap workflows definitions into the dev container
-func mountWorkflowDefConfigMapMutateVisitor(cm *v1.ConfigMap) mutateVisitor {
+// mountDevConfigMapsMutateVisitor mounts the required configMaps in the Workflow Dev Deployment
+func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVisitor {
 	return func(object client.Object) controllerutil.MutateFn {
 		return func() error {
 			deployment := object.(*appsv1.Deployment)
 			volumes := make([]v1.Volume, 0)
 			volumeMounts := make([]v1.VolumeMount, 0)
 
-			volumes = append(volumes, v1.Volume{
-				Name: configMapWorkflowDefVolumeNamePrefix,
-				VolumeSource: v1.VolumeSource{
-					ConfigMap: &v1.ConfigMapVolumeSource{
-						LocalObjectReference: v1.LocalObjectReference{Name: cm.Name},
+			volumes = append(volumes,
```

Comment:
In this function we are considering volumes empty, is better to save the volumes present, because if someone in the future call this function after the external resources we wipe all the volumes present in the deployment.